apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/page.tsx

```diff
@@ -60,101 +72,89 @@ export default async function Pool({
   );
 
   return (
-    <div className="relative mx-auto flex max-w-7xl gap-3 px-4 sm:px-6 lg:px-8">
-      <div className="flex flex-1 flex-col gap-6 rounded-xl border-2 border-black bg-surface p-16">
-        <header className="flex flex-col items-center justify-center">
-          <h2 className="text-center font-press">Pool {poolId} </h2>
+    <div className="mx-auto flex max-w-5xl flex-col gap-10">
+      {/* Header */}
+      <section className="section-layout flex flex-col gap-0 overflow-hidden">
+        <header>
+          <h2>
+            Pool #{poolId} - {title}
+          </h2>
         </header>
-        <main className="flex flex-col gap-10">
-          {/* Description section */}
-          <section className="relative flex w-full flex-col items-center overflow-hidden rounded-lg border-2 border-black bg-white">
-            <div className="mt-4 flex w-full flex-col items-center gap-12 p-8">
-              <h3 className="max-w-2xl  text-center font-semibold">{title}</h3>
-              {!isEnabled && (
-                <div className="badge badge-warning absolute left-5 top-5 gap-2 p-4 font-bold">
-                  Pending review from community council
-                </div>
-              )}
+        <p className="mb-2">
+          <EthAddress address={strategyAddr} />
+        </p>
+        <p>{description}</p>
+        <div className="mb-7 mt-5 flex w-full flex-col items-start gap-3.5">
```

Comment:
mb in this section is 40px = mb-10
mt is 32px
gap is 8px

no need for w-full (this is by default in div tags)